docs/_posts/examples/3400-01-13-filter-markers.html

```diff
@@ -136,11 +136,6 @@
 });
 
 map.on('load', function() {
-    // Add a GeoJSON source containing place coordinates and information.
-    map.addSource("places", {
```

Comment:
This example adds multiple layers using this source; keep the explicit `addSource`.